aexpect/ops.py

```diff
@@ -60,6 +60,14 @@
 LOG = logging.getLogger(__name__)
 
 
+def _process_status_output(command, status, output):
+    if output != "":
+        output = output.strip()
+    if status != 0:
+        raise ShellCmdError(command, status, output)
+    return status, output
```

Comment:
Given the status will always be 0, do you think it is necessary to report it back?

Reply:
The idea here is simply to have a consistent interface with a "brief pass over the status and output" where some common processing could raise errors and mutate any of the two argument. It is true only some such processing is done right now but I think this could more easily generalize would we need further processing of both.